### PR TITLE
[cmake] fix ldflags transfer on addon depends build

### PR DESCRIPTION
**Pull request by AlwinEsch**
## Description

Before was only a "-DCMAKE_EXE_LINKER_FLAGS=-L${OUTPUT_DIR}/lib" given.
But by cross compile on some depends are library linker flags needed who give
infos about the OS (specially ffmpeg).

This change add the give of them in same way as already CFLAGS and CPPFLAGS
are given.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by AlwinEsch** [comment]
jenkins build this with addons please

File: cmake/scripts/common/HandleDepends.cmake [hunk 1]
```diff
@@ -78,8 +78,10 @@ function(add_addon_depends addon searchpath)
           # make sure we create strings, not lists
           set(TMP_C_FLAGS "${CMAKE_C_FLAGS} ${ARCH_DEFINES}")
           set(TMP_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ARCH_DEFINES}")
+          set(TMP_EXE_LINKER_FLAGS "-L${OUTPUT_DIR}/lib ${CMAKE_EXE_LINKER_FLAGS} ${ARCH_DEFINES}")
```
Comment (Rechi) [inline cmake/scripts/common/HandleDepends.cmake]: Why are define flags needed in the linker flags?

Comment (AlwinEsch) [reply cmake/scripts/common/HandleDepends.cmake]: Make no sense, was only a nightly c&p from above and my Mr. Brainbrain was on other side of world.

jenkins build this with addons please

**Review by phunkyfish — APPROVED** [review]
LGTM.

**Comment by AlwinEsch** [comment]
jenkins build this with addons please

**Pull request merged**